exchange/exchange.go

```diff
@@ -41,11 +44,13 @@ type bidResponseWrapper struct {
 	bidder       openrtb_ext.BidderName
 }
 
-func NewExchange(client *http.Client, cfg *config.Configuration) Exchange {
+func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *config.Configuration) Exchange {
 	e := new(exchange)
 
 	e.adapterMap = newAdapterMap(client, cfg)
 	e.adapters = make([]openrtb_ext.BidderName, 0, len(e.adapterMap))
+	e.cache = cache
+	e.cacheTime = time.Duration(cfg.CacheURL.ExpectedTimeMillis) * time.Millisecond
```

Comment:
if this is not set in the config, cache time will be zero?

Reply:
correct. Do you think that makes sense as a default, or would it be better to have some time here?

Per [the note here](https://github.com/prebid/prebid-server/pull/276/files#diff-3baf47c64847a8fb8aaa8cc2e088513bR131), I hope this doesn't become a long-term solution in any case.

Reply:
seems fine to me, just wanted to make sure that was what we wanted.